news/2024/2024-12-14-osu-world-cup-2024-concludes.md

```diff
@@ -0,0 +1,585 @@
+
+How much do you consider tryouts when forming the roster as opposed to tournament performance throughout the year? How about the previous year's OWC performance?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/9224078" style="background-image: url('/wiki/shared/avatars/FlyingTuna.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/9224078">FlyingTuna</a></p>
+
+Last year, our main focus was on whether we could compete skillsets like speed and mechanics which favor USA when we face them in the later rounds.
+
+However, this year, instead of focusing on skillsets that favor USA, we decided to focus on skillsets that we are good at (like tech, alt, hidden, gimmick) and just give up on speed.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3958619" style="background-image: url('/wiki/shared/avatars/dio.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/3958619">D I O</a></p>
+
+You have [Nopekjk](https://osu.ppy.sh/users/4585186) listed as an analyst on your team collab, and [Allegrissimo](https://osu.ppy.sh/users/9052194) shouted him out with smoke writing during the TB showcase. What was his role this year as an analyst and past player for OWC South Korea?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/9224078" style="background-image: url('/wiki/shared/avatars/FlyingTuna.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/9224078">FlyingTuna</a></p>
+
+Yes, Nopekjk was our analyst. Throughout the whole OWC he would analyze the scores that the opponents would get on each map of each stage and would help us with pick and ban whilst also encouraging us and supporting us mentally.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3958619" style="background-image: url('/wiki/shared/avatars/dio.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/3958619">D I O</a></p>
+
+The Grand Final had a turning point when South Korea was down 4–1 early on. How did your team react, and what changed in order for you to win?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/9224078" style="background-image: url('/wiki/shared/avatars/FlyingTuna.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/9224078">FlyingTuna</a></p>
+
+My team and I, we all took it calmly as we had the bracket reset in mind. Because no matter what happens in the first part, even if we fail we had that 1 more chance. So we all remained calm and positive and even if we were down 4-1 we were just as confident and strong as ever because we are used to reverse sweeping.
```

Comment:
```suggestion
My team and I, we all took it calmly as we had the bracket reset in mind. Because no matter what happens in the first part, even if we fail we had that one more chance. So we all remained calm and positive and even if we were down 4-1 we were just as confident and strong as ever because we are used to reverse sweeping.
```

Reply:
eh